_sass/hovers.scss

```diff
@@ -0,0 +1,165 @@
+
+// Converted Variables
+
+
+// Custom Media Query Variables
+
+
+/*
+
+  HOVER EFFECTS
+  Docs: http://tachyons.io/docs/themes/hovers/
+
+    - Dim
+    - Glow
+    - Hide Child
+    - Underline text
+    - Grow
+    - Pointer
+    - Shadow
+
+*/
+
+/*
+
+  Dim element on hover by adding the dim class.
+
+*/
+.dim-2 {
```

Comment:
You can remove anything that isn't custom from this file – anything copied from Tachyons that you didn't change. No need to repeat it since it's already in Tachyons.